pipeline/clinvar/clinvar.py

```diff
@@ -21,6 +21,20 @@ def textIfPresent(element, field):
     else:
         return(ff.text.encode('utf-8'))
 
+
+def processClinicalSignificanceElement(el, obj):
```

Comment:
I can see the logic in this being a separate function since both `referenceAssertion` and `clinVarAssertion` need it, but this seems like it should be a method of the class since it just sets fields in the object you pass in. Perhaps `referenceAssertion` and `clinVarAssertion` could inherit from an `assertion` base class that includes this function as a method?

Reply:
Yes, that's a good point! For me,  `referenceAssertion` and `clinVarAssertion` are not too similar, despite their name, therefore I prefer this composition approach. But I'll be touching this code in a subsequent iteration and might take your suggestion up again.